README.md

```diff
@@ -1 +1,36 @@
-Snowplow Data Models Repo
+![snowplow-logo](snowplow_logo.jpg)
+
+Snowplow is a scalable open-source platform for rich, high quality, low-latency data collection. It is designed to collect high quality, complete behavioural data for enterprise business.
+
+# Snowplow Pipeline Overview
+
+![snowplow-pipeline](snowplow_pipeline.png)
+
+The [Snowplow trackers](https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/) enable highly customisable collection of raw, unopinionated event data. The pipeline validates these events against a JSONSchema - to guarantee a high quality dataset - and adds information via both standard and custom enrichments.
+
+This data is then made available in-stream for real-time processing, and can also be loaded to blob storage and data warehouse for analysis.
+
+The SQL data models in this repo aggregate those raw events in-warehouse, to produce consumable datasets for analysis and actionable insight to be built upon. The models are built to allow the addition of custom use cases and aggregations, in a robust, scalable way.
+
+# Prerequisites
+
+These models are written in a format that is runnable via [SQL-runner](https://github.com/snowplow/sql-runner) - available for download from [Bintray](https://dl.bintray.com/snowplow/snowplow-generic/).
```

Comment:
Add a more direct link to sql-runner on bintray?

Reply:
This one's an annoying little detail. Two things: 1. Direct link is platform specific. 2. Direct link hardcodes version (and there's no 'latest' - so I'm conscious of it getting out of date/adding to a list of things that need to be manually managed if we have a sql-runner release.

Thoughts on those?

Reply:
Link here instead: https://bintray.com/snowplow/snowplow-generic/sql-runner#files ? (with or without #files in the url)